source/win-spout-output.cpp

```diff
@@ -11,26 +11,24 @@
 #include <util/threading.h>
 #include "win-spout.h"
 
-#include "SpoutLibrary.h"
```

Comment:
Interesting - do you think this might have had something to do with the crash earlier?

Reply:
@campbellwmorgan no, not at all I had the same crash in filers even without this include - I removed here because it wasn't being used :-)